src/ui/flutter_app/lib/shared/custom_drawer/src/header.dart

```diff
@@ -48,25 +41,19 @@ class CustomDrawerHeader extends StatelessWidget {
       LocalDatabaseService.colorSettings.drawerHighlightItemColor,
     ];
 
-    final animation = GestureDetector(
-      onDoubleTap: _enableDeveloperMode,
-      child: AnimatedTextKit(
-        animatedTexts: [
-          ColorizeAnimatedText(
-            title,
-            textStyle: TextStyle(
-              fontSize: LocalDatabaseService.display.fontSize + 16,
-              fontWeight: FontWeight.w600,
-            ),
-            colors: _animatedTextsColors,
-            speed: const Duration(seconds: 3),
-          ),
-        ],
-        pause: const Duration(seconds: 3),
-        repeatForever: true,
-        stopPauseOnTap: true,
-        onTap: () => debugPrint("$_tag DoubleTap to enable developer mode."),
-      ),
+    final animation = AnimatedTextKit(
+      animatedTexts: [
+        ColorizeAnimatedText(
+          title,
+          textStyle: AppTheme.drawerHeaderTextStyle,
+          colors: _animatedTextsColors,
+          speed: const Duration(seconds: 3),
```

Comment:
I remember the original time value is 30 seconds, otherwise it is too fancy, it is just an easter egg.

Reply:
Yeah it was written in milisecs and I might habe misread the number. That's why I converted it to seconds jn the first place (more readable code) 

Fix will come